Command/KlarnaShippingCommand.php

```diff
@@ -63,53 +65,75 @@ public function execute(InputInterface $input, OutputInterface $output)
             ]);
         }
 
-        if (
-            $transactions !== null
-            && is_array($transactions)
-        ) {
+        if ($transactions !== null && is_array($transactions)) {
+
             $output->writeln(count($transactions) . ' orders to update.');
-     
+
             foreach ($transactions as $transaction) {
-                $output->writeln('Updating order ' . $transaction->getOrderNumber());
 
-                /** @var Order $order */
-                $order = null;
-             
-                if ($transaction->getOrderId() === null) {
-                    continue;
-                }
+                // Ship order
+                try {
 
-                /** @var Repository $orderRepository */
-                $orderRepository = $this->modelManager->getRepository(Order::class);
+                    $output->writeln('>> updating order: ' . $transaction->getOrderNumber());
 
-                if ($orderRepository === null) {
-                    continue;
-                }
+                    /** @var Order $order */
+                    $order = null;
 
-                /** @var Order $order */
-                $order = $orderRepository->find($transaction->getOrderId());
+                    if ($transaction->getOrderId() === null) {
+                        continue;
+                    }
 
-                if ($order === null) {
-                    continue;
-                }
+                    /** @var Repository $orderRepository */
+                    $orderRepository = $this->modelManager->getRepository(Order::class);
+
+                    if ($orderRepository === null) {
+                        continue;
+                    }
+
+                    /** @var Order $order */
+                    $order = $orderRepository->find($transaction->getOrderId());
+
+                    if ($order === null) {
+                        continue;
+                    }
+
+
+                    $switcher = new MollieShopSwitcher($this->container);
```

Comment:
Will it actually be necessary to (re)create that newable at any of the foreach loop's iterations? Or may the switcher instantiation be placed outside the loop?